frontend/src/routes/Config.svelte

```diff
@@ -367,14 +366,36 @@
 					})
 					break
 				case 'reports':
-					$exportConfig['items'] = ['inspections','reports']
-					ExportReports()
-					push("/export/status")
+					ValidateExportDirectory().then((result) => {
+						if (result === true) {
+							showBanner = false
```

Comment:
can we wrap this code in a function and maybe pass a flag to do the associated export?

Reply:
we can. But wanted to limit the risk for now. We can definitely optimize this as a fix-it-friday thing